contractcourt/briefcase.go

```diff
@@ -473,7 +473,6 @@ func (b *boltArbitratorLog) FetchUnresolvedContracts() ([]ContractResolver, erro
 				if err := timeoutRes.Decode(resReader); err != nil {
 					return err
 				}
-				timeoutRes.AttachResolverKit(resKit)
```

Comment:
here we can instead call `res.Decode` after the switch.

Reply:
that is still possible in this commit, but not anymore when `decode` becomes a constructor